sdk/daml-lf/snapshot/src/benchmark/scala/com/digitalasset/daml/lf/testing/snapshot/ReplayBenchmark.scala

```diff
@@ -53,7 +53,7 @@ class ReplayBenchmark {
     )
     if (darFile.nonEmpty) {
       val loadedPackages = TransactionSnapshot.loadDar(Paths.get(darFile))
-      benchmark = benchmark.adapt(loadedPackages)
+      benchmark = benchmark.copy(pkgs = loadedPackages)
```

Comment:
Dropped the use of the `Adapter` as it looks like we do not need to perform any package ID rewrites at this moment in time.

We will likely remove `Adapter` completely in a future PR?

Reply:
Will do this as on issue [Remove deprecated snapshot/Adapter.scala](https://github.com/digital-asset/daml/issues/21609)